### PR TITLE
flashlist ⚡ ⚡ ⚡ 

### PR DESCRIPTION
**Pull request by intergalacticspacehighway**
# Why
- Recyclerlistview's API is bad. (too much hardcoding is needed)
- Recyclerlistview has issues with viewable events on android. 
- FlashList is working super smooth for feed

# How
- Replace recyclerlistview with FlashList for swipe-list component

# Test Plan
- Test feed/swipe-list on android, iOS and web

### CODE REVIEW

File: packages/app/hooks/use-infinite-list-query.ts [hunk 1]
```diff
@@ -39,7 +39,7 @@ export const useInfiniteListQuerySWR = <T>(
     isLoading,
   } = useSWRInfinite<T>(urlFunction, fetcher, {
     revalidateFirstPage: true,
-    suspense: true,
+    // suspense: true,
```
Comment (axeldelafosse) [inline packages/app/hooks/use-infinite-list-query.ts:42]: We might want to revisit this later

**Review by axeldelafosse — APPROVED** [review]
LFG ✨

**Review by alantoa — APPROVED** [review]
LFG! 🔥

File: packages/app/hooks/use-infinite-list-query.ts (hunk 1 above)
Comment (intergalacticspacehighway) [reply packages/app/hooks/use-infinite-list-query.ts:42]: added it back. I think this solves it. https://github.com/showtime-xyz/showtime-frontend/pull/1346/files#diff-8bad9b025a5c54b5d6ca8cd87337cb619e02274a68ceb62d22d4959dacc92d76R50

There's some part of me that doesn't like it very much. 😖

Comment (intergalacticspacehighway) [reply packages/app/hooks/use-infinite-list-query.ts:42]: really makes it hard to do stuff like show the stale content along with a  loading spinner.  Show the error along with the stale content

Comment (intergalacticspacehighway) [reply packages/app/hooks/use-infinite-list-query.ts:42]: reverted it, the list was flashing (literal flash-list) due to suspense. Will revisit!
https://github.com/showtime-xyz/showtime-frontend/pull/1346/commits/e5d7c6ce1d4ff9c9c047103d087c12ccae73e831

**Pull request merged**